bibtex-actions.el

```diff
@@ -235,11 +235,33 @@ offering the selection candidates"
              collect (cdr (or (assoc choice candidates)
                               (rassoc choice candidates))))))
 
-(defun bibtex-actions--format-candidates ()
-  "Transform candidates from 'bibtex-completion-candidates'.
+(defun bibtex-actions--normalize-paths (file-paths)
+  "Return a list of FILE-PATHS normalized with truename."
+  (if (stringp file-paths)
+      ;; If path is a string, return as a list.
+      (list (file-truename file-paths))
+    (delete-dups (mapcar (lambda (p) (file-truename p)) file-paths))))
+
+(defun bibtex-actions--local-files-to-cache ()
+  "The local bibliographic files not included in the global bibliography."
+  ;; We cache these locally to the buffer.
+  (let* ((local-bib-files
+          (bibtex-actions--normalize-paths
+           (bibtex-completion-find-local-bibliography)))
+         (global-bib-files
+          (bibtex-actions--normalize-paths
+           bibtex-completion-bibliography)))
+    (seq-difference local-bib-files global-bib-files)))
+
+(defun bibtex-actions--format-candidates (&optional context)
+  "Format candidates, with optional hidden CONTEXT metadata.
 This both propertizes the candidates for display, and grabs the
 key associated with each one."
-  (let* ((main-template
+  (let* ((bibtex-completion-bibliography
```

Comment:
We also need to remove the let-binding on `bibtex-completion-bibliography` here. SIce it will override the binding in refresh function.

Reply:
Done.